### ISSUE TITLE
install ubuntu 20.04

### ISSUE THREAD

**Issue by david739** [Bitcoin-ABC/bitcoin-abc#490]
hi can someone please give me a working installation procedure. I tried this post failed

sudo apt-get install bsdmainutils build-essential libssl-dev libevent-dev lld ninja-build python3

sudo apt-get install apt-transport-https ca-certificates gnupg software-properties-common wget
wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc 2>/dev/null | sudo apt-key add -

sudo apt update
sudo apt install cmake

git clone https://github.com/Bitcoin-ABC/bitcoin-abc.git

cd bitcoin-abc

mkdir build
cd build
cmake -GNinja ..
ninja
ninja install

cmake -GNinja ..

-- Looking for stddef.h
-- Looking for stddef.h - found
-- Check size of __int128
-- Check size of __int128 - done
-- Found PkgConfig: /usr/bin/pkg-config (found version "0.29.1")
CMake Error at /usr/share/cmake-3.16/Modules/FindPackageHandleStandardArgs.cmake:146 (message):
  Could NOT find Jemalloc (missing: Jemalloc_INCLUDE_DIR) (Required is at
  least version "3.6.0")

      Reason given by package: this is an optional dependency that can be disabled by passing -DUSE_JEMALLOC=OFF to the cmake command line

Call Stack (most recent call first):
  /usr/share/cmake-3.16/Modules/FindPackageHandleStandardArgs.cmake:393 (_FPHSA_FAILURE_MESSAGE)
  cmake/modules/FindJemalloc.cmake:111 (find_package_handle_standard_args)
  src/CMakeLists.txt:289 (find_package)

-- Configuring incomplete, errors occurred!
See also "/home/pool/bitcoin-abc/build/CMakeFiles/CMakeOutput.log".
See also "/home/pool/bitcoin-abc/build/CMakeFiles/CMakeError.log".

**Comment by Fabcien** [comment]
If you don't need to build a static binary (and you probably don't), you need to install the other dependencies as per the instructions: https://github.com/Bitcoin-ABC/bitcoin-abc/blob/master/doc/build-unix.md, see:
`Now, you can either build from self-compiled [depends](https://github.com/Bitcoin-ABC/bitcoin-abc/blob/master/depends/README.md) or install the required dependencies with the following instructions.`.

But since you are running Ubuntu 20.04 there is an easier solution, installing via our PPA: https://launchpad.net/~bitcoin-abc/+archive/ubuntu/ppa/

**Comment by david739** [comment]
Error 
sudo add-apt-repository ppa:bitcoin-abc/ppa
sudo: add-apt-repository: command not found

**Comment by david739** [comment]
there really isn't anyone here who can get bitcoin-abc working

**Comment by Fabcien** [comment]
I pointed you to the instructions in the above comment. Did you install all the dependencies as indicated in the doc ? That's why your build fails, because you don't have the dependencies.

**Comment by david739** [comment]
yes i tried it doesn't work we have modified the procedure above as we did the installation. same error can you please send me a working procedure or edit mine

**Comment by Fabcien** [comment]
The procedure in your first post is still missing most of the dependencies, you need to install them. Eventually you can disable them if you're sure you don't need them.
The actual error tells you that you're missing jemalloc. From that same document, you can install it via:
`sudo apt-get install libjemalloc-dev`
or disable it by adding `-DUSE_JEMALLOC=OFF` to the cmake command line.

Just do the same for each dependency listed in the document.
